src/style.css

```diff
@@ -1 +1,56 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  margin: 0;
+}
+
+.search-form {
+  margin-top: 20px;
+}
```

Comment:
According to the checklist, it's recommended to use either top or bottom margin for consistency and to avoid potential margin collapse issues. Consider using only `margin-bottom` for elements and removing `margin-top`.